draft-irtf-cfrg-vdaf.md

```diff
@@ -5098,10 +5098,40 @@ possible. See {{xof-vs-ro}} for more information.
 | FieldLeaf  | `Field255` ({{fields}}) |
 {: #idpf-bbcggi21-param title="Constants and type definitions for the IDPF of BBCGGI21."}
 
-### Key Generation
+### Overview
+
+On a high level, the IDPF maps a key generator's input `(alpha, beta_1, ..., beta_BITS)` onto
+a binary tree with `BITS+1` levels, where each edge going from a parent node to a left child is
+labeled `0`, and each right edge is labeled `1`. Then each leaf node corresponds to a value
+`x` in `0, ..., 2^BITS`, and the labels on the path from the root to `x` contain the binary
+decomposition of `x`, starting with the most significant bit. Finally, all inner nodes have
+the label `0`, while the nodes on the path from the root to `alpha` have labels
```

Comment:
We are overloading the concept of "labels" here. The first half of the paragraph says that the labels are from successive bits of the IDPF index, while the second half says that the labels are zero or beta. Should we refer to the latter as "values", and not "labels"? Moreover, I don't think the distinction "inner nodes" needs to be made, since both inner nodes and leaf nodes will be labeled similarly.

Reply:
Done.